src/tox/tox_env/python/virtual_env/package/pyproject.py

```diff
@@ -214,7 +214,10 @@ def perform_packaging(self, for_env: EnvConfigSet) -> list[Package]:
         elif of_type == "sdist":
             self.setup()
             with self._pkg_lock:
-                sdist = self._frontend.build_sdist(sdist_directory=self.pkg_dir).sdist
+                print("cc", self.core["config_settings"])
```

Comment:
I don't think this is part of the core settings, and I don't think it should be. This is the packaging environment setting in my view.

Reply:
I'm not really sure what this means. What part of the code is the _packaging env setting_?

Reply:
Got it.